frontend/src/Components/ProductListingPage/SortFunction.js

```diff
@@ -0,0 +1,34 @@
+
+const sortByDiscount = (products) => {
+    const sortedProducts = products.sort((a,b) => {
```

Comment:
You can implement this in single line as done in `lowToHighSort` below

Reply:
done